authors/apps/authentication/tests/test_registration.py

```diff
@@ -43,7 +43,8 @@ def test_send_verification_mail(self):
         self.assertEqual(mail.outbox[0].subject, "Activate your account.")
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
-    def test_empty_username_field(self):
+
```

Comment:
The test is failing because of some changes you made here. Please check.

Reply:
Check on this `IndentationError: unindent does not match any outer indentation level`